blockmanager.go

```diff
@@ -954,8 +954,10 @@ func (b *blockManager) getCheckpointedCFHeaders(checkpoints []*chainhash.Hash,
 				log.Warnf("Banning peer=%v for invalid "+
 					"checkpoints", sp)
 
-				b.server.BanPeer(sp)
-				sp.Disconnect()
+				go func() {
```

Comment:
Can be squashed with the original commit that introduced banning.

Reply:
That first commit adds banning new peers, not ignoring peers that we've already banned.